docs/pages/server-access/guides/teleport-ssh.mdx

```diff
@@ -0,0 +1,241 @@
+
+```yaml
+request:
+  roles:
+  - dba
+  - admin
+```
+
+Users can then request a role through `tsh`:
+
+1. A user logins into `teleport-cluster` and requests the `admin` role: `tsh login teleport-cluster --request-roles=admin`.
+2. This will generate a unique request token identifying the request, requested role, and user.
+
+  - `Seeking request approval... (id: bc8ca931-fec9-4b15-9a6f-20c13c5641a9)`
+
+3. A Teleport admin can approve or deny their request from within the Teleport Web UI or through [tctl](../../cli-docs.mdx#tctl-request-ls):
+
+  - `tctl request ls`
+  - `tctl request approve [token]`
+  - `tctl request deny [token]`
+
+As such, Teleport allows organizations to include complex [access request workflows](../../enterprise/workflow/index.mdx) directly within their primary SSH and authentication tool.
+
+## Introspect and manage the cluster 
+
+Teleport `tctl` ("tea-kettle") has powerful querying and multi-command capabilities to make inspecting and managing a cluster even easier.
+
+1. Teleport admins can define `labels` for any Node to easily group resources beyond cluster management.
+
+2. These [labels](../../admin-guide.mdx#labeling-nodes-and-applications) can be used to query any resource sharing them: 
+
+   - `tsh ssh root@env=example` returns the list of all resources with the `env=example` label specified in their `teleport.yaml` (`'env': 'example'`).
+
+Suppose an IP address changes. An admin can quickly find the current Node with that label since it remains unchanged.
+
+Suppose you need to query the SSH Node of each cluster. Give each SSH Node the same label so a single query will allow you to introspect specific resources in multiple clusters.
+
+### Multi-querying
+
+`tctl` also supports multi-querying: the ability to execute sequential (or simultaneous) commands on multiple resources at once. 
+
+   - `tsh ssh root@env=example ls` will execute the `ls` command on each Node displaying the contents of each to the screen.
+
+As a consequence, Teleport admins can execute commands on all Nodes that share a label vastly simplifying repeated operations. 
+
+### Progammatic interaction 
+
+Teleport Auth Nodes expose an API through gRPC to allow for [programmatic interaction](../../reference/api.mdx) with its underlying functionalities beyond the command-line.
+
+Teleport clients communicate with clusters by connecting to Teleport Auth Nodes and authenticate using short-lived certificates and Teleport [roles](../../access-controls/reference.mdx).
+
+This requires:
+
+1. A suitable client written in your favorite language.
+2. Provisioning valid TLS (SSL) certificates.
+3. Supplying the correct roles, permissions, and/or short-lived certificates to the appropriate users.
+
+Review the `client` [pkg.go reference documentation](https://pkg.go.dev/github.com/gravitational/teleport/api/client) for more information about working with the Teleport API programmatically.
+
+## Configure short-lived certificates for access
```

Comment:
Same here, I would factor this into a separate guide like here:

https://goteleport.com/docs/kubernetes-access/guides/cicd/